redbot/core/core_commands.py

```diff
@@ -2177,6 +2177,80 @@ async def command_manager(self, ctx: commands.Context):
         """Manage the bot's commands."""
         pass
 
+    @checks.is_owner()
+    @command_manager.command(name="defaultdisablecog")
+    async def command_default_disable_cog(self, ctx: commands.Context, *, cogname: str):
+        """ Set the defaulft state for a cog as disabled. """
+        cog = self.bot.get_cog(cogname)
+        if not cog:
+            return await ctx.send(_("No such cog"))
+        if cog == self:
+            return await ctx.send(_("You can't disable this cog by default."))
+        await self.bot._disabled_cog_cache.default_disable(cogname)
+        await ctx.send(_("{cogname} has been set as disabled by default.").format(cogname=cogname))
+
+    @checks.is_owner()
+    @command_manager.command(name="defaultenablecog")
+    async def command_default_enable_cog(self, ctx: commands.Context, *, cogname: str):
+        """ Set the defaulft state for a cog as enabled. """
+        cog = self.bot.get_cog(cogname)
+        if not cog:
+            return await ctx.send(_("No such cog"))
+        await self.bot._disabled_cog_cache.default_enable(cogname)
+        await ctx.send(_("{cogname} has been set as enabled by default.").format(cogname=cogname))
+
+    @commands.guild_only()
+    @command_manager.command(name="disablecog")
+    async def command_disable_cog(self, ctx: commands.Context, *, cogname: str):
+        """ Disable a cog in this guild. """
+        cog = self.bot.get_cog(cogname)
+        if not cog:
+            return await ctx.send(_("No such cog"))
+        if cog == self:
+            return await ctx.send(_("You can't disable this cog as you would lock yourself out."))
+        if await self.bot._disabled_cog_cache.disable_cog_in_guild(cogname, ctx.guild.id):
+            await ctx.send(_("{cogname} has been disabled in this guild.").format(cogname=cogname))
+        else:
+            await ctx.send(
+                _("{cogname} was already disabled (nothing to do).").format(cogname=cogname)
+            )
+
+    @commands.guild_only()
+    @command_manager.command(name="enablecog")
+    async def command_enable_cog(self, ctx: commands.Context, *, cogname: str):
+        """ Enable a cog in this guild. """
+        if await self.bot._disabled_cog_cache.enable_cog_in_guild(cogname, ctx.guild.id):
+            await ctx.send(_("{cogname} has been enabled in this guild.").format(cogname=cogname))
+        else:
+            # putting this here allows enabling a cog that isn't loaded but was disabled.
+            cog = self.bot.get_cog(cogname)
+            if not cog:
+                return await ctx.send(_("No such cog"))
```

Comment:
A bit nit-picky suggestion:
```suggestion
                return await ctx.send(_("Cog with the given name doesn't exist."))
```